R/error-checks.R

```diff
@@ -25,21 +27,17 @@ isOfType <- function(object, type, nullAllowed = FALSE) {
 
   type <- .typeNamesFrom(type)
 
-  inheritType <- function(x) {
```

Comment:
No, this breaks the behavior. E.g., 

```
ds1 <- ospsuite::DataSet$new(name = "ds1")
ds2 <- ospsuite::DataSet$new(name = "ds2")
tmp <- list(ds1, NULL, ds2)

isOfType(tmp, ospsuite::DataSet, nullAllowed = TRUE)
[1] FALSE
```

In the `develop` branch it returns `TRUE`, which is expected.